playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -106,7 +106,8 @@ This should return a 301 Found response **not** a connection refused error.
 
 <a name="ios"></a>
 ### iOS ###
-1. Download [Shadowrocket](https://itunes.apple.com/us/app/shadowrocket/id932747118?mt=8) and launch it.
+1. Download [Shadowrocket](https://itunes.apple.com/us/app/shadowrocket/id932747118?mt=8) and launch it.    
+> **Note:** Shadowrocket is not available in the China App Store. Apple was forced to remove it due to Chinese Government regulations. If you are using the China App Store and cannot find Shadowrocket, try switching to another country's App Store first.
```

Comment:
I'm not familiar enough with the ecosystem to know how easy it is to change your app store country. I suspect that doing that can cause other problems and I'm hesitant to recommend that as a solution without more understanding of the steps required & potential side effects. 

> If you guys prefer the shorter than that's cool, can you go ahead and edit my PR?

Ok